src/api/app/models/decision.rb

```diff
@@ -24,6 +27,10 @@ def create_event
   def event_parameters
     { id: id, moderator_id: moderator.id, reason: reason, reportable_type: reports.first.reportable.class.name }
   end
+
+  def track_decision
+    RabbitmqBus.send_to_bus('metrics', "decision,kind=#{kind},hours_before_decision=#{((updated_at - created_at) / 1.hour).floor} count=1")
+  end
```

Comment:
```suggestion
  def track_decision
    hours_before_decision = ((updated_at - created_at) / 1.hour).floor
    RabbitmqBus.send_to_bus('metrics', "decision,kind=#{kind},hours_before_decision=#{hours_before_decision} count=1")
  end
```

Reply:
Yeah, I will take this out and put it in a function. I was somewhat divided about this when submitting the PR.